README.md

```diff
@@ -312,6 +313,10 @@ Please note that `url` or `connection` can't be used at the same time.
 
 * `queue_size`: the size (maximum) of a queue which is considered unhealthy (the default is 100).
 
+### Solr
+
+* `url`: the URL used to connect to your Solr instance - must be a string. You can also use `url` to explicitly configure authentication (e.g., `'http://user:pass@example.solr.com:8983/'`)
+
```

Comment:
Could you update the example to `https`? 😅

Reply:
Done!